testing/trino-testing-services/src/main/java/io/trino/testng/services/ReportAfterMethodNotAlwaysRun.java

```diff
@@ -64,7 +64,7 @@ public void onBeforeClass(ITestClass testClass)
     }
 
     @VisibleForTesting
-    static void checkHasAfterMethodsNotAlwaysRun(Class<?> testClass)
+    static void checkHasAfterMethodsAlwaysRun(Class<?> testClass)
```

Comment:
> The point of the method is to make sure that there are no methods annotated with without alwaysRun

Actually, it checks if there are any `@AfterX` method which have `alwaysRun = false`, hence "NotAlwaysRun". Perhaps it could be less confusing, though...